src/components/Kanban/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { DragDropContext, Droppable } from "react-beautiful-dnd";
+import Table from "../Table";
+import LogBar from "../LogBar";
+import { useSelector } from "react-redux";
+import { RootState } from "../../store";
+import { addButton, card, messageWrapper, wrapperKanban, wrapperTables } from "./style.css";
+import { useState } from "react";
+import AddInput from "../AddInput";
+import {
+  addTable,
+  addLog,
+  changeTableOrder,
+  changeTaskOrder,
+} from "../../store/newBoardSlice";
+import { useDispatch } from "react-redux";
+
+function Kanban() {
+  const [addTableVar, setAddTable] = useState(false);
+  const board = useSelector((state: RootState) => state.newBoardSlice.selected);
+  const tables = useSelector(
+    (state: RootState) => state.newBoardSlice.selected?.tables
+  );
+  const dispatch = useDispatch();
+
+  function handleOnDragEnd(result: any) {
+    if (result.type === "COLUMN") {
+      dispatch(
+        changeTableOrder({
+          from: +result.source.index,
+          to: +result.destination.index,
+        })
+      );
+    } else {
+      const to_table = +result.destination.droppableId.split("-")[1];
+      const from_table = +result.source.droppableId.split("-")[1];
```

Comment:
Please use `camelCase` for variable names 
If a backend uses `snake_case` map required values manually just before sending / right after fetching the data